docs/manual/installation/system-requirements.de.md

```diff
@@ -17,7 +17,7 @@ gepflegten Versionen von Contao sind mit den neuesten PHP- und MySQL-Versionen k
 empfohlen, diese immer zu verwenden.
 
 - **PHP:** Version 7.3+ (neueste Patch-Version)
-- **MySQL:** Version 5.7+ oder gleichwertiger **MariaDB** Server
+- **MySQL:** Version 5.7+ aber vor 8.014 oder gleichwertiger **MariaDB** Server
```

Comment:
I would instead insert a notice warning of the current incompatibility:

```markdown
{{% notice warning %}}
Derzeit besteht eine Inkompatibilität mit MySQL Version **8.0.17** und MariaDB Version **XYZ**.
{{% /notice %}}
```
Please research the exact MariaDB versions as well, where the new reserved keywords were introduced.